app/services/foods_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodsService
+  def get_url(url)
+    conn = Faraday.new("https://api.nal.usda.gov/fdc/v1/foods/")
```

Comment:
Typically the faraday connection can just have the base url (everything up until .gov) and then the path can be in the get call